packages/rest/src/rest.server.ts

```diff
@@ -233,6 +225,24 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
+  /**
+   * Set up CORS middleware
+   */
+  protected _setupCORS() {
+    // Allow CORS support for all endpoints so that users
+    // can test with online SwaggerUI instance
+    const corsOptions = this.config.cors || {
+      origin: '*',
+      methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
+      preflightContinue: false,
+      optionsSuccessStatus: 204,
+      maxAge: 86400,
+      credentials: true,
+    };
+    // Set up CORS
+    this._expressApp.use(cors(corsOptions));
```

Comment:
Can we register this using `this._registerMiddlerware()` to serve as an example.